modules/002-deckhouse/templates/deployment.yaml

```diff
@@ -175,6 +176,8 @@ spec:
               value: /tmp/shell-operator-debug.socket
             - name: HISTFILE
               value: /tmp/.bash_history
+            - name: DEBUG_HTTP_PORT
+              value: "9652"
```

Comment:
We also have to add this variable [here](https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/kubernetes/actions/manifests/manifests.go#L320)

Reply:
Done